tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Redis/ServiceStack/RedisNativeClientSendReceiveIntegration.cs

```diff
@@ -19,7 +19,7 @@ namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.Redis.ServiceStack
         AssemblyName = "ServiceStack.Redis",
         TypeName = "ServiceStack.Redis.RedisNativeClient",
         MethodName = "SendReceive",
-        ReturnTypeName = "T",
+        ReturnTypeName = "!!0",
```

Comment:
oh it must have been fun updating all these 🤣

Reply:
:nod-yes: